lib/full-site-editing/edit-site-page.php

```diff
@@ -223,6 +223,16 @@ function register_site_editor_homepage_settings() {
 			'description'  => __( 'The ID of the page that should be displayed on the front page', 'gutenberg' ),
 		)
 	);
+
+	register_setting(
+		'reading',
+		'page_for_posts',
+		array(
+			'show_in_rest' => true,
+			'type'         => 'number',
+			'description'  => __( 'The ID of the page that should display the latest posts', 'gutenberg' ),
+		)
+	);
```

Comment:
I noticed that static page settings aren't available when the plugin is deactivated. I think we forgot to expose these settings in REST API for 5.9.